### DISCUSSION TITLE
Lower the minimum donate amount?

### DISCUSSION THREAD

**Discussion by anonymousaga** [photoprism/photoprism#4234] [General]
Twice or three times a year, I pick out my top few open-source softwares for that season and donate $5-$10 to the projects. This time, I wanted to donate to photoprism. 
However, I noticed the $25 minimum one-time donation on github sponsors, and I'm not comfortable donating $25. I don't care about any perks; I just want to donate to keep the project alive. I'm also not interested in keeping track of a monthly subscription for 3-5 months to donate $10.

Have you considered lowering the donate threshold to get more donations? Thanks :)

**Comment by lastzero** [comment]
We appreciate your feedback! There are two reasons why the minimum donation amount on GitHub Sponsors currently is $25:

- Manual work is needed on our end to book each donation based on where it is coming from, so that we can pay the right amount of taxes as required by law in Germany. This is probably easier / less work for maintainers based in the US.
- Most sponsors wanted to have one of the benefits listed below and some didn't notice these have different pricings, so they donated a custom amount and then wondered why they don't get an email with instructions. Setting the limit to the lowest tier with benefits was the easiest way to avoid this.

Would it be alright if we lowered the minimum amount to $10?

**Comment by anonymousaga** [comment]
Yeah that would be excellent!

**Reply by graciousgrey** [reply]
We have just updated the minimum amount :)

**Comment by anonymousaga** [comment]
Great! Donated! Thanks for the great project!
